cmd/vic-machine/create/create.go

```diff
@@ -1610,6 +1610,11 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
Also we need to check for the alias name also not having any spaces, or we'll have the same problem again.